### PR TITLE
add new routes and fix imports

### PR DESCRIPTION
**Pull request by hectorguerrini**

### CODE REVIEW

**Comment by VgsStudio** [comment]
LGTM

File: iac/iac/lambda_stack.py [hunk 1]
```diff
@@ -39,5 +39,7 @@ def __init__(self, scope: Construct, mss_student_api_resource: Resource) -> None
         self.createLambdaApiGatewayIntegration(module_name="create_selfie", method="POST", mss_student_api_resource=mss_student_api_resource)
         self.createLambdaApiGatewayIntegration(module_name="update_selfie", method="POST", mss_student_api_resource=mss_student_api_resource)
         self.createLambdaApiGatewayIntegration(module_name="delete_selfie", method="POST", mss_student_api_resource=mss_student_api_resource)
+        self.createLambdaApiGatewayIntegration(module_name="get_all_selfies", method="GET", mss_student_api_resource=mss_student_api_resource)
+        self.createLambdaApiGatewayIntegration(module_name="get_all_students", method="GET", mss_student_api_resource=mss_student_api_resource)
```
Comment (VgsStudio) [inline iac/iac/lambda_stack.py:44]: organizar selfies e e studentes separadas

**Review by JoaoVitorBranco — COMMENTED** [review]
Boa, grande mago!

**Review by JoaoVitorBranco — APPROVED** [review]
Boa, grande mago!

**Pull request merged**